src/selfupdater.cpp

```diff
@@ -164,10 +165,11 @@ void SelfUpdater::startUpdate()
 
   auto latestRelease = m_UpdateCandidates.begin()->second;
 
-  QMessageBox query(QMessageBox::Question,
-                    tr("New update available (%1)")
-                        .arg(latestRelease["tag_name"].toString()), tr("Do you want to install update? All your mods and setup will be left untouched.\nSelect Show Details option to see the full change-log."),
-                    QMessageBox::Yes | QMessageBox::Cancel, m_Parent);
+  UpdateDialog dialog(m_Parent);
+  dialog.setVersions(
+    MOShared::createVersionInfo().displayString(3),
+    latestRelease["tag_name"].toString()

```

Comment:
I would use the release name instead of the tag name, it's a bit nicer (also below).

Reply:
Changed the details expander to use the release name.  Tag name makes more sense here to me as it's going to be directly compared to a version string.  It would be a little weird to say "You currently have 2.4.0 but 2.5.3 Really Fixed This Time is available".  Not that anyone would name a release that but they could.